buildpacks/jupyterlab/bin/jupyterlab-entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+JUPYTER_ENV_DIR=$1
+
+if [ -z "$JUPYTER_ENV_DIR" ]; then
+	echo "WARNING: Running with JUPYTER_ENV_DIR not set at all, this means python and jupyter executables are expected at /bin/."
+fi
+
+if [ -n "$RENKU_WORKING_DIR" ]; then
+	if [ ! -d "$RENKU_WORKING_DIR" ]; then
+		mkdir -p "$RENKU_WORKING_DIR"
+	fi
+	# This allows the terminal in jupyterlab to open in the working directory
+	cd "$RENKU_WORKING_DIR"
```

Comment:
This is the fix. All else is just moving the entrypoint in its own file. When the entrypoint is its own file it can be linted with shellcheck.